web/rootfs/defaults/settings-config.js

```diff
@@ -224,6 +229,15 @@ config.peopleSearchQueryTypes = ['user','conferenceRooms'];
 // Prejoin page.
 config.prejoinPageEnabled = {{ $ENABLE_PREJOIN_PAGE }};
 
+// Welcome page.
+config.enableWelcomePage = {{ .Env.ENABLE_WELCOME_PAGE }};
```

Comment:
Use the defined variables so $ENABLE_WELCOME_PAGE

Reply:
Oh my bad, I will change that.

Reply:
Done, I changed the 3 I added.